prime-angular-frontend/src/app/core/models/pagination.model.ts

```diff
@@ -1,11 +1,6 @@
-export class Pagination {
-  public page?: number;
-  public perPage?: number;
-  public total?: number;
-
-  constructor({ page, perPage: perPage }: Partial<Pagination> = { page: 1, perPage: 20 }) {
-    this.page = page || 1;
-    this.perPage = perPage || 20;
-    this.total = 0;
-  }
+export interface Pagination {
+  page: number;
+  totalPages: number;
+  total: number;
```

Comment:
Can a comment be added for `total`?

Reply:
Or it be renamed to `totalRecords` or something

Reply:
Went a similar path, changed it to totalResults in the front and backend